File: frontend/src/api/h5Config.ts
import http from './http'
import type { H5Config } from './types'

export const defaultH5Config: H5Config = {
  siteTitle: '教材购书服务',
  siteSubtitle: '个人信息与教材订购入口',
  serviceWechat: '请联系管理员',
  servicePhone: '待配置',
  workTime: '09:00 - 18:00',
  noticeText: '下单后请保持电话畅通，配送信息以后续通知为准。',
}

export function getH5Config() {
  return http.get<unknown, H5Config>('/h5/config')
}

File: backend/src/main/java/com/example/wechatstore/modules/h5config/entity/H5Config.java
package com.example.wechatstore.modules.h5config.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@TableName("h5_config")
public class H5Config {

    @TableId(type = IdType.AUTO)
    private Long id;
    private String siteTitle;
    private String siteSubtitle;
    private String serviceWechat;
    private String servicePhone;
    private String workTime;
    private String noticeText;
    private LocalDateTime createTime;
    private LocalDateTime updateTime;
}
